extension/chrome/elements/compose-modules/compose-err-module.ts

```diff
@@ -152,6 +152,10 @@ export class ComposeErrModule extends ViewModule<ComposeView> {
   };
 
   public throwIfEncryptionPasswordInvalid = async ({ subject, pwd }: { subject: string, pwd?: string }) => {
```

Comment:
We should rename this method to fully reflect its function, e.g. `throwIfEncryptionPasswordInvalidOrDisabled`
Otherwise, all looks goof to me 👍

Reply:
Sounds good